tiles/src/main/java/com/protomaps/basemap/Basemap.java

```diff
@@ -137,15 +139,21 @@ static void run(Arguments args) {
       .addGeoPackageSource("landcover", sourcesDir.resolve("daylight-landcover.gpkg"),
         "https://r2-public.protomaps.com/datasets/daylight-landcover.gpkg");
 
-    Downloader.create(planetiler.config()).add("ne", neUrl, nePath).run();
+    Downloader.create(planetiler.config()).add("ne", neUrl, nePath)
+      // .add("pgf-encoding", "https://api.github.com/repos/wipfli/pgf-encoding/zipball/e9c03fb", sourcesDir.resolve("main.zip"))
```

Comment:
Is the idea here the hash refers to a specific compilation of Unicode PUA -> precomposed glyph mappings, and if the hash changes, that means a major version bump and the encodings break? Is there an opportunity to SemVer those?

Is the hash version also associated with a set of Fontstacks (one for each supported script)?

Reply:
I removed the commit has altogether and added some comments to the semver markdown file.

* The basemap major version has to bump when we change the version of a script that we already include
* The basemap minor version has to bump when we add a new script